README.md

```diff
@@ -2,31 +2,39 @@
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Discuss abstraction in object-oriented programming
+- Compare abstract classes and interfaces
 
 ## Introduction
 
-A 1-2 sentence summary of what will be covered.
 
-## Topic 1
+A programmer uses abstraction to hide all but the essential details about an object in an effort
+to reduce complexity and increase adaptability.  Abstraction give us the ability to separate **what** should
+happen from **how** it should happen.
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+Abstraction can be achieved in Java by defining interfaces and abstract classes,
+along with their concrete class implementations.
 
-## Topic 2
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+## Comparing Abstract Classes and Interfaces
 
-## Conclusion
+- An abstract class may define instance and static variables that are inherited by subclasses.
+- An interface can define constants variables (static and final), but not instance or non-final static variables.
+- Neither an interface nor an abstract class can be instantiated using the `new` operator.
+- An abstract class can have parameterized constructors and the default constructor is always present in an abstract class.
+  - Abstract class constructors are called by subclass constructors to initialize inherited state.
+- An interface can't have a constructor.
+- An abstract class may define any number of concrete or abstract methods, including instance and static methods.
+- An interface may define any number of abstract, default, or static methods.
```

Comment:
Suggestion: Maybe a table showing the differences might be easier to visualize?

Reply:
good idea, updated lesson.